configure.ac

```diff
@@ -1546,6 +1551,35 @@ if test "$enable_threads" = "multiple" ; then
 fi
 AC_DEFINE_UNQUOTED([MPICH_THREAD_GRANULARITY],$thread_granularity,[Method used to implement atomic updates and access])
 
+# Check for enable-vci-method choice
+vci_method=MPICH_VCI__ZERO
+case $enable_vci_method in
+    zero)
+    # Essentially single VCI
+    vci_method=MPICH_VCI__ZERO
+    ;;
+    communicator)
+    # Every communicator gets a new VCI in a round-robin fashion
+    vci_method=MPICH_VCI__COMM
+    ;;
+    tag)
+    # Explicit VCI info embedded with a tag scheme (5 bit src vci, 5 bit dst vci, plus 5 bit user)
+    vci_method=MPICH_VCI__TAG
+    ;;
+    implicit)
+    # An automatic scheme taking into account of user hints
+    vci_method=MPICH_VCI__IMPLICIT
+    ;;
+    explicit)
+    # Directly passing down vci in parameters (MPIX or Endpoint rank)
+    vci_method=MPICH_VCI__EXPLICIT
```

Comment:
`Endpoints` don't explicitly specify VCIs. They specify communicator handles which could map to different VCIs or the same. Anyways, since these aren't standard yet, how about just remove them?

Reply:
In the endpoint proposal, `MPI_COMM_CREATE_ENDPOINT(parent_comm, my_num_ep, info, new_comm)` will create a communicator with `my_num_ep * num_processes` ranks. Within that communicator, a rank uniquely identifies the process and vci.

Reply:
Yes, that's what we would do internally. What I meant to say was that the user doesn't deal with VCIs but with communicator handles and ranks, in terms of naming.